frontend/src/routes/Home/Search/Details/YAMLPage.tsx

```diff
@@ -247,9 +247,10 @@ export function EditorActionBar(props: {
                         <Button
                             variant="secondary"
                             id="download-resource-button"
+                            icon={<DownloadIcon />}
                             onClick={() => downloadYaml(name, resourceYaml)}
                         >
-                            <DownloadIcon /> {t('public~Download')}
```

Comment:
@zlayne Trying to access the `public` translation namespace broke our automatic updating of our translations file.

Reply:
Thanks for the catch here.